cmd/maya-apiserver/app/server/pool_endpoint_v1alpha1.go

```diff
@@ -0,0 +1,85 @@
+package server
```

Comment:
Can we add the `Copyright ...` comment.

Reply:
other endpoints are not having it should we add it ?

Reply:
At least add for yours..
Others we can take it later.

Reply:
done